client/src/components/DropdownPace/DropdownPace.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+const Select = styled.select`
+  height: 40px;
```

Comment:
Be careful setting absolute height or width, because changing the global font size (or font) might crash the design.
Better calculate the height based on the content plus padding or control it in the parent.

Reply:
Part of my refactoring issue